tests/endpoints/test_cargo_movements_real.py

```diff
@@ -79,3 +80,16 @@ def test_search_list_filter_id(self):
 
         assert len(df) == 2
         print(tabulate.tabulate(df))
+
+    def test_speed(self):
+        with Timer("Search") as t:
+            cms = CargoMovements().search(
+                filter_time_min="2019-08-29T00:00:00.000Z",
+                filter_time_max="2019-08-30T00:00:00.000Z",
+            )
+
+        with Timer("to_list") as t:
+            list = cms.to_list()
```

Comment:
`list` is a built-in function name, and should not be overwritten.  This could be called `list_of_cms`